src/examples/java/io/nats/examples/jetstream/NatsJsUtils.java

```diff
@@ -218,7 +218,7 @@ public static byte[] makeData(String prefix, int msgSize, boolean verbose, int x
             return null;
         }
 
-        String text = prefix + "-" + x + ".";
+        String text = prefix + "-" + x;
```

Comment:
Just some example stuff, not production code, always tweaking things because I'm obsessive